### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ashokirla**

### CODE REVIEW

**Review by sanjeev9160 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -213,22 +215,52 @@ However in the following rare circumstances you might run into difficulties:
...
-The following are the notifications available as part of "deployments" feature:
-1. Deployment review pending notifications for your environments being deployed through GitHub Actions workflow.
-<p align="left"><img width="500" alt="Review Pending" src="docs/images/ReviewPending.png"></p>
+You can unsubscribe for workflows notifications by just running the below command:
+`/github unsubscribe org/repo workflows`
 
-2. Deployment review completed notifications for your environments being deployed through GitHub Actions workflow.
-<p align="left"><img width="500" alt="Approval completed" src="docs/images/ReviewCompleted.png"></p>
+#### Updates to the app permissions
+To receive the above notifications you need to grant access for Actions events for Microsoft Teams in your GitHub organization. You will be prompted to do so as you try to enable `workflows` feature for your organization the first time.
```
Comment (anuraag016) [inline README.md:249]: Says Microsoft teams

Comment (ashokirla) [reply README.md]: Removed it.

**Review by anuraag016 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -213,22 +215,53 @@ However in the following rare circumstances you might run into difficulties:
...
+By default, when you configure workflow notifications without passing any filters, it is configured for workflows triggered via pull requests targeting your default branch.
+You can pass one or multiple entries. 
+
+You can unsubscribe for workflows notifications by just running the below command:
+`/github unsubscribe org/repo workflows`
+
+#### Updates to the app permissions
+To receive the above notifications you need to grant access to receive Actions events by GitHub app in Slack. You will be prompted to do so as you try to subscribe for `workflows` feature for your organization the first time.
+
+<p align="left"><img width="500" alt="Workflow notification" src="docs/images/ActionsPermissions.png"></p>
```
Comment (shogo82148) [inline README.md:251]: Hi, I've just received the ActionsPermissions message.
I understand that I need read permissions, but I don't understand why I need write permissions. Could you please clarify this point?

Comment (uncoverd) [reply README.md:251]: I'm guessing the action buttons require write access.

Screenshot 2022-12-02 at 13 49 08

Comment (shogo82148) [reply README.md:251]: @ashokirla Is it right?

Comment (sanjeev9160) [reply README.md:251]: @shogo82148 Yes, the app needs write permissions to do certain actions on your behalf. Like Re-run all jobs, Approve/Reject, Cancel workflow, Re-run failed jobs.

Comment (ejmuentes) [reply README.md:251]: I would prefer to be able to opt in to the write-dependent features

**Pull request merged**